R/rdataretriever.R

```diff
@@ -419,8 +417,13 @@ get_citation = function(dataset) {
 }
 
 #' Reset the scripts or data(raw_data) directory or both
-#'
 #' @param scope All resets both  scripst and data directory
+#' @export
+#' @import reticulate
+#' @examples
+#' \donttest{
+#' rdataretriever::reset('iris')
+#' }
 reset = function(scope = 'all') {
   r_data_retriever = reticulate::import('retriever')
```

Comment:
@harshitbansal05, What happened was that we did add the roxygen comment "doc string" but did not include ` @export`. 
In this case, reset was not exported to name space.
 
When a function changes, we should install the new changes and `roxygenise` the package

```
library(DBI)
library(RPostgreSQL)
library(RSQLite)
library(reticulate)
library(RMariaDB)
install.packages(".", repos = NULL, type="source")
roxygen2::roxygenise()
devtools::test()
```